packages/core/src/actions/assembleExternalQuote.ts

```diff
@@ -27,7 +32,13 @@ export async function assembleExternalQuote(
   config: AuthConfig,
   parameters: AssembleExternalQuoteParameters,
 ): Promise<AssembleExternalQuoteReturnType> {
-  const { quote, updatedOrder, doGasEstimation = false } = parameters
+  const {
+    quote,
+    updatedOrder,
+    doGasEstimation = false,
+    useGasSponsorship = false,
+    refundAddress = zeroAddress,
```

Comment:
There isn't any possibility we send funds to the null address accidentally right?

Reply:
no, in the contract we interpret the zero address to mean "send funds back to `tx.origin`"